src/bot/.gitignore

```diff
@@ -0,0 +1,138 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
How does multiple
.gitignore files work?
Do they apply to the subdirectories?

Reply:
They do